### ISSUE TITLE
Improve documentation of page xliff converter

### ISSUE THREAD

**Issue by timoludwig** [Integreat/cms-django#332]
Explain how the page xliff converter works and add docstrings of the following format:
```
"""
[Summary]

:param [ParamName]: [ParamDescription], defaults to [DefaultParamVal]
:type [ParamName]: [ParamType](, optional)
...
:raises [ErrorType]: [ErrorDescription]
...
:return: [ReturnDescription]
:rtype: [ReturnType]
"""
```

**Comment by svenseeberg** [comment]
Will be included in #341

**Comment by svenseeberg** [comment]
Closed with cf73f4cf09850335e535aea450f98759e5f2b819
